README.md

```diff
@@ -338,46 +350,69 @@ Append the first activity to occur after the primary activity.
+> activities. Consider using **First In Between** instead.
+
 ### Last After ([source](./macros/relationships/append_only/last_after.sql))
 Append the last activity to occur after the primary activity.
 
 **Dataset Usage:**
 - `primary_activity:` ❌
 - `appended_activity:` ✅
 
-**Example Usage:** For the **First Ever** 'visited_website' append **Last
+**Example Usage:**
+
+For the **First Ever** 'visited_website' append **Last
 After** 'returned_item. The most recent time a customer returned an item after
 their first visit.
 
-### First Before ([source](./macros/relationships/append_only/first_before.sql))
-Append the first activity to occur before the primary activity.
+> **Note:** Be catious when using this with **All Ever** for the primary activity.
+> It will result in adding the same **Last After** activity to multiple primary
+> activity records, if the appended activity occurred after multiple primary
+> activities. Consider using **Last In Between** instead.
+
+### First In Between ([source](./macros/relationships/append_only/first_in_between.sql))
+Append the first activity to occur after each occurrence of the primary
+activity, but before the next occurrence of the primary activity.
 
 **Dataset Usage:**
 - `primary_activity:` ❌
 - `appended_activity:` ✅
 
 **Example Usage:**
 
-For every 'visited_website' append **First Before** 'opened_email'. This will
-add the the first email that the customer opened before their first visit.
+For **All Ever** 'visited_website' append **First In Between** 'completed_order'. On
+every website visit, did the customer order before the next visit. (generally
+used for event-based conversion)
 
-### First In Between ([source](./macros/relationships/append_only/first_in_between.sql))
-Append the first activity to occur after each occurrence of the primary
-activity, but before the next occurrence of the primary activity.
+> **Note:** The appended activity will be added to the row of the final occurance of the
+> primary activity, even though it is not technically _in between_ another occurance of the
+> primary activity. The generated SQL for the dataset can be filtered further if
+> desired, to remove those rows.
+
+### Last In Between ([source](./macros/relationships/append_only/last_in_between.sql))
+Append the last activity that occurred after each occurrence of the primary
+activity and before the next occurrence of the primary activity.
 
 **Dataset Usage:**
 - `primary_activity:` ❌
 - `appended_activity:` ✅
 
 **Example Usage:**
 
-For every 'visited_website' append **First In Between** 'completed_order'. On
-every website visit, did the customer order before the next visit. (generally
-used for event-based conversion)
+For **All Ever** 'visited_website' append **Last In Between** 'viewed_page'. On every
+website visit, what was the last page that they viewed before leaving.
+
+> **Note:** The appended activity will be added to the row of the final occurance of the
```

Comment:
Same note as in [L394](https://github.com/tnightengale/dbt-activity-schema/pull/14/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R394)